src/components/Menus/Sidebar/DownloadPanel/DownloadPanel.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import SidebarPanel from "../SidebarPanel";
+import { useTranslation } from "react-i18next";
+import DownloadButton from "../../../DownloadButton/DownloadButton";
+import DesignSystemButton from "../../../DesignSystemButton/DesignSystemButton";
+import DownloadIcon from "../../../../assets/icons/download.svg";
+import "../../../../assets/stylesheets/DownloadPanel.scss";
+
+export const DownloadPanel = () => {
+  const { t } = useTranslation();
+
+  return (
+    <SidebarPanel heading={t("downloadPanel.heading")}>
+      <div className="download-panel__content">
+        <div className="download-panel__subtitle">
+          {t("downloadPanel.subtitle")}
+        </div>
+        <p>{t("downloadPanel.logInHint")}</p>
+        <div className="download-panel__button-container">
+          <DesignSystemButton
+            className="btn btn--primary download-panel__button"
+            text={t("downloadPanel.logInButton")}
+            type="primary"
+            onClick={() => {
+              console.log("Log in clicked");
+            }}
+          />
+          <DesignSystemButton
+            className="btn btn--secondary download-panel__button"
+            text={t("downloadPanel.signUpButton")}
+            type="secondary"
```

Comment:
Has no effect on the style but does seem to enable hover behaviour

Reply:
its not actually using the design system styles is it, so none of [these](https://github.com/RaspberryPiFoundation/design-system-core/blob/b8037974fccdb7ea413fe5bb871f6e13b9180442/scss/components/button.scss#L84-L119) but instead using the old button styles [here](https://github.com/RaspberryPiFoundation/editor-ui/blob/0f5f19aaa50b44cf6307924e6633e03bf4fe07ac/src/assets/stylesheets/Button.scss#L182)